packages/taxdat/R/covariate_helpers.R

```diff
@@ -1392,16 +1396,35 @@ get_country_admin_units <- function(iso_code,
                     shapeType = paste0("ADM", admin_level))%>% 
       dplyr::select(shapeName = !!rlang::sym(paste0("name_", admin_level)),
                     shapeID,
-                    shapeType)
-    sf::st_crs(boundary_sf) <- sf::st_crs(4326)
-    message("Using the aggregated gadm shapefiles for this region")
-  } else if(iso_code %in% c("COD","BDI","ETH","MWI","UGA")&admin_level==0){
-    boundary_sf <- rgeoboundaries::gb_adm0(iso_code) %>%
-      dplyr::select(shapeName,shapeID,shapeType,geometry)
-    sf::st_crs(boundary_sf) <- sf::st_crs(4326)
+                    shapeType) %>% 
+      dplyr::mutate(source = "gadm")
+    
+  } else if(iso_code %in% c("COD","BDI","ETH","MWI","UGA") & admin_level == 0){
```

Comment:
@QLLZ I think we could make this a bit more concise using the geoboundaries function in rgeoboundaries https://github.com/wmgeolab/rgeoboundaries/blob/master/R/geoboundaries.R -- then we won't need different if conditions for each admin level